libraries/stdlib/src/kotlin/random/Random.kt

```diff
@@ -267,10 +267,12 @@ public abstract class Random {
      *
      * @sample samples.random.Randoms.defaultRandom
      */
-    companion object Default : Random() {
-
+    companion object Default : Random(), Serializable {
+        private const val serialVersionUID = 6449517406788167720L
         private val defaultRandom: Random = defaultPlatformRandom()
```

Comment:
Better to use `writeReplace` to avoid exposing `defaultRandom` field.

Reply:
What should `writeReplace` return then?

Reply:
@ilya-g

Reply:
Just an empty serializable class whose `readResolve` returns `Random.Default`.

Reply:
On the other hand, we could rely on the fact that companion object doesn't have any fields and keeps all its state as static in the containing class, but IMO that's a too fragile assumption.

Reply:
Then I'll make defaultRandom a property without backing field, since *ThreadLocalRandom is an `object`.